src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Service/CreateServiceCommandHandler.php

```diff
@@ -71,37 +68,17 @@ public function handle(CreateServiceCommand $command): void
         $service->setOrganizationNameEn($command->getOrganizationNameEn());
         $service->setOrganizationNameNl($command->getOrganizationNameNl());
         $this->serviceRepository->isUnique($service);
-        $this->serviceRepository->save($service);
-    }
-
-    private function createTeamData(string $name, string $teamName, string $email): array
-    {
-        $emails = $this->createEmailsArray($email);
 
-        return [
-            'name' => $teamName,
-            'description' => $this->translator->trans(
-                'teams.create.description',
-                [
-                '%teamName%' => $name,
-                ]
-            ),
-            'personalNote' => $this->translator->trans('teams.create.personalNote'),
-            'viewable' => false,
-            'emails' => $emails,
-            'roleOfCurrentUser' => 'MANAGER',
-            'invitationMessage' => $this->translator->trans('teams.create.invitationMessage'),
-            'language' => 'ENGLISH',
-        ];
-    }
+        $response = $this->inviteRepository->createRole(
+            "$serviceName {$command->getOrganizationNameEn()}",
+            "$serviceName {$command->getOrganizationNameEn()}",
+            $roleDescription,
+            $this->landingUrl,
+            $this->spDashboardManageId,
+        );
 
-    private function createEmailsArray(string $email): array
-    {
-        $emails = [];
-        foreach (explode(',', $email) as $mail) {
-            $emails[trim($mail)] = 'MANAGER';
-        }
+        $service->registerInvite($response->urn, $response->id);
 
-        return $emails;
+        $this->serviceRepository->save($service);
```

Comment:
This is still an open end right? You are going to use the `CreateRoleResponse` to invite the admin user to this 'team' in a next dev-effort?

Reply:
Yes, thats the next ticket Bart prepared.